cwf/gateway/go.mod

```diff
@@ -11,7 +11,6 @@
 //
 module magma/cwf/gateway
 
-// TODO remove golang.org/x/net line once Go Upgrade (https://github.com/magma/magma/pull/12151) is merged
```

Comment:
Shouldn't the replacement of x/net be removed here @alexzurbonsen? To me it looks like you've removed the comment but not the relevant line.

Reply:
Good catch.